nginx/templates/_gzip.jinja

```diff
@@ -0,0 +1,29 @@
+{%- if gzip is defined and gzip is mapping %}
+{% from "nginx/lib.sls" import configurable %}
+  # Start configure gzip for vhost
+  gzip {{ 'on' if gzip.enable|default(true) else 'off' }};
```

Comment:
Can we first test this on a new server before highstating it to every vhost? Setting the default op `false` should fix this.